tests/qos/qos_sai_base.py

```diff
@@ -1872,3 +1872,18 @@ def dualtor_ports_for_duts(request, get_src_dst_asic_and_duts):
         logger.info("Finish fetching dual ToR info {}".format(dualtor_ports_set))
 
         return dualtor_ports_set
+
+    @pytest.fixture(autouse=False)
+    def _check_ingress_speed_gte_400g(
```

Comment:
@rraghav-cisco, can you please elaborate more on why this testcase needs to be skipped for 400G? does this platform not support pg drop counters?

Reply:
@neethajohn We do support pg drop counter in 400G ports.

With HBM, for 400G longlink ports, we have 90MB PFC threshold and 90MB headroom and that requires over 2 million 64B packets to trigger ingress drop.
We cannot support 2M 64B packets, since just over 1M 64B packets takes only about 68MB memory in HBM and fills up all the HBM blocks for one voq in HBM.

So PGDrop test must be skipped when we have 400G longlink port as ingress.